content/docs/04.5-devx/08-registries.md

```diff
@@ -14,109 +14,23 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 
 
-# Palette Helm Registry
-Helm Charts are a collection of Kubernetes resource files capable of deploying services ranging in varying complexities. Palette provides a few stable public Helm registries out of the box. Tenants can also add any public or private Helm registries to leverage charts. Palette makes deploying typical Kubernetes applications easier with native support for Helm charts. We also allow the addition of any public or private Helm chart registry to the system. Charts from these system registries or any additional ones added by tenant administrators are modelled into cluster profiles for cluster deployment.
+# Palette Dev Engine Registries
 
-Palette mandates using Helm OCI version 3.7.0 and above for the best security experience.
+The pack registry is a server-side application to store and serve packs to its clients. Packs from a pack registry are retrieved and presented as options during the creation of a cluster profile. Palette supports the configuration of multiple registries.
 
-#  Deploying Helm Charts
-Palette allows the deployment of Helm charts in two modes, one using the default Public Repo and another leveraging developers's custom Helm registry.
+## Default Registry
+The default pack registry is Spectro Cloud's public pack registry. It consists of several packs that make it easy for a user to quickly create a cluster profile and launch a Kubernetes cluster with their choice of integrations. Palette maintains all packs in this pack registry and takes care of upgrading packs in the pack registry whenever required.
```

Comment:
```suggestion
The default pack registry is Spectro Cloud's public pack registry. It consists of several packs that make it simple for a user to quickly create a cluster profile and launch a Kubernetes cluster with their choice of integrations. Spectro Cloud maintains all packs in the default pack registry, this includes taking care of upgrades in the pack registry whenever required.
```
Avoid using the word "easy", use "simple" instead. Easy refers to something that does not require a great effort or that does not present lot of difficulties. Simple refers to something that is direct, straightforward not complex.